uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/manage-leave-requests.spec.js

```diff
@@ -1,929 +1,953 @@
+            describe('when location filter has no value', function () {
+              beforeEach(function () {
+                controller.filters.contact.location = null;
+                controller.refresh();
+              });
 
-                it('Contact API gets called with null', function () {
-                  expect(Contact.leaveManagees).toHaveBeenCalledWith(controller.contactId, jasmine.objectContaining({
-                    level_type: null
-                  }));
-                });
+              it('ContactAPI gets called with null', function () {
+                expect(Contact.leaveManagees).toHaveBeenCalledWith(controller.contactId, jasmine.objectContaining({
+                  location: null
+                }));
               });
             });
           });
 
-          describe('for admin', function () {
-            beforeEach(function () {
-              role = 'admin';
-              compileComponent();
-            });
-
-            describe('region', function () {
-              describe('when region filter has value', function () {
-                var mockRegion = 'mockvalue';
+          describe('levelTypes', function () {
+            describe('when levelTypes filter has value', function () {
+              var mockLevelType = 'mockvalue';
 
-                beforeEach(function () {
-                  controller.filters.contact.region = mockRegion;
-                  controller.refresh();
-                });
+              beforeEach(function () {
+                controller.filters.contact.level_type = mockLevelType;
+                controller.refresh();
+              });
 
-                it('Contact API gets called with region value', function () {
-                  expect(Contact.all).toHaveBeenCalledWith(jasmine.objectContaining({
-                    region: mockRegion
-                  }));
-                });
+              it('ContactAPI gets called with level types value', function () {
+                expect(Contact.leaveManagees).toHaveBeenCalledWith(controller.contactId, jasmine.objectContaining({
+                  level_type: mockLevelType
+                }));
               });
+            });
 
-              describe('when region filter has no value', function () {
-                beforeEach(function () {
-                  controller.filters.contact.region = null;
-                  controller.refresh();
-                });
+            describe('when levelTypes filter has no value', function () {
+              beforeEach(function () {
+                controller.filters.contact.level_type = null;
+                controller.refresh();
+              });
 
-                it('Contact API gets called with null', function () {
-                  expect(Contact.all).toHaveBeenCalledWith(jasmine.objectContaining({
-                    region: null
-                  }));
-                });
+              it('Contact API gets called with null', function () {
```

Comment:
same.

ummmmmm, are we testing the same thing over and over?

Please update the wording wherever it appears (there are still a few more to go).